pkg/operator/ceph/object/controller.go

```diff
@@ -203,12 +213,24 @@ func (r *ReconcileCephObjectStore) reconcile(request reconcile.Request) (reconci
 	// DELETE: the CR was deleted
 	if !cephObjectStore.GetDeletionTimestamp().IsZero() {
 		logger.Debugf("deleting store %q", cephObjectStore.Name)
+		cephObjectStore.Spec.PreservePoolsOnDelete = false
```

Comment:
Was this only for testing? I assume we're not committing this change to always purge the pools.